src/hotspot/share/gc/shared/genArguments.hpp

```diff
@@ -33,6 +33,8 @@ extern size_t MinNewSize;
 extern size_t MinOldSize;
 extern size_t MaxOldSize;
 
+extern size_t OldSize;
```

Comment:
Any reason we still want to keep `OldSize` variable? becase GCs really care about are `init`, `min` and `max` values.

Reply:
The concept of "initial old-gen size" will always be there. If `OldSize` is removed, all readers need to be updated to `InitialHeapSize - NewSize`. It's not obvious that is definitely better/more readable.